src/static/js/modules/show-hide-fields.js

```diff
@@ -0,0 +1,41 @@
+/* ==========================================================================
+ * Show/Hide Additional Fields
+ * This is for form pages on Office templates.
+ * Shows additional fields on forms when additional information is needed.
+ *
+ * Set the id for fieldset with additional fields with the data-attribute
+ * `data-additional-fields`
+ *
+ * ========================================================================== */
+
+'use strict';
+
+var $ = require( 'jquery' );
+
+function init() {
+
+  function toggleAdditionalFields( input ) {
```

Comment:
- Move function to top-level, it shouldn't be nested in `init`.
- Re-name `_toggleAdditionalFields`.
- Pass in reference to `this` to avoid need for closure.